fastlane/lib/fastlane/actions/docs/sync_code_signing.md

```diff
@@ -407,6 +407,16 @@ fastlane match change_password
 
 You'll be asked for the new password on all your machines on the next run.
 
+### Import
+
+To import and encrypt a certificate (`.cer`) and the private key (`.p12`) into the _match_repo run:
```

Comment:
```suggestion
To import and encrypt a certificate (`.cer`) and the private key (`.p12`) into the _match_ repo run:
```

Reply:
I read through this like 10000 times and I didn't catch this 😝 I'm so bad at doc